src/docs/development/add-to-app/android/project-setup.md

```diff
@@ -4,4 +4,211 @@ short-title: Integrate Flutter
 description: Learn how to integrate a Flutter module into your existing Android project.
 ---
 
-TODO(blasten): document.
\ No newline at end of file
+Flutter can be embedded into your existing Android application in a piecemeal
+fashion as a source code sub-project or as AARs.
+
+The integration flow can be done via the Android Studio IDE with the
+[Flutter plugin](https://plugins.jetbrains.com/plugin/9212-flutter) or manually.
+
+## Using Android Studio
+
+The Android Studio IDE offers a convenient way of integrating your Flutter
+module automatically. With Android Studio, you can co-edit both your Android
+code and your Flutter code in the same project. You can also continue to use
+your normal IntelliJ Flutter plugin functionalities such as Dart code completion,
+hot reload, widget inspector etc.
+
+Add-to-app flows with Android Studio is only supported on Android Studio 3.6+
```

Comment:
"3.6+" is technically not correct today as 4.0 is not supported by the plugin. We're working on it though.

Reply:
Ah, thanks. Good to know. Should I just say 3.6 specifically for the moment?

Reply:
sgtm